### ISSUE TITLE
word-to-word highlight for multiple lines change

### ISSUE THREAD

**Issue by dai-shi** [rtfpessoa/diff2html#1]
Hi, thanks for this nice library. I found something that is different from the GitHub style.
It's about word-to-word highlight (BTW, the README says char-to-char, which actually is word-to-word, isn't it)

If you have multiple lines changed, this doesn't work.
The example input is the following:

```
diff --git a/foo.js b/foo.js
---
+++
@@ -1,4 +1,5 @@
+zero
 one
-two word
-three word
+to word
+tree word
 four
```

It would be so nice if this is supported.

**Comment by rtfpessoa** [comment]
@dai-shi if you have any ideas how to do this I would be glad to accept a pull request.

**Comment by rtfpessoa** [comment]
@dai-shi take a look at #7 

And try it with [diff2html-cli](https://www.npmjs.com/package/diff2html-cli)

I made diff in blocks and bot char-by-char and word-by-word.

Any feedback?

**Comment by dai-shi** [comment]
Just did a quick test (not with the cli, but directly), it's awesome! So far working as expected.
https://github.com/dai-shi/codeonmobile/commit/03b08a1c853a5b34cf85fef368d3ad698098b98e
Wonderful work!

**Comment by rtfpessoa** [comment]
fixed by #7
